en_US/cluster-linking/quick-start.md

```diff
@@ -0,0 +1,160 @@
+# Setup Cluster Link
+
+This page provides a quick start guide to set up a Cluster Link between two distant EMQX clusters.
+
+## Prerequisites
+
+Suppose you have compute resources in two different regions and you want to have a EMQX cluster in each region, with the ability to link the two clusters together. Let's say the regions are `us-east` and `eu-west`, thus the clusters are named `cluster-us-east` and `cluster-eu-west` respectively.
+
+This is easily achievable if those clusters:
+1. Are running EMQX 5.8.0 or later.
+2. Have distinct, unique Cluster names.
+3. Are able to communicate with each other over the network.
+
+Last point is crucial. Cluster Linking requires both `cluster-us-east` MQTT listener to be reachable from the `cluster-eu-west` network, and vice versa, otherwise the link will not be fully operational. While not strictly necessary, it is strongly recommended to put those MQTT listeners behind a load balancer, to ensure even distribution of Cluster Linking protocol traffic.
+
+For simplicity, suppose also that the clusters will talk to each other over a VPN or similar mechanism, essentially taking care of most of the security concerns, thus simplifying the cluster configuration.
```

Comment:
Maybe worth a brief mentioning that configuring SSL is possible and highly recommended otherwise? 
Though, this would probably be obvious in future if we have Dashboard UI to setup cluster links...